packages/story-editor/src/app/story/effects/useLoadStory.js

```diff
@@ -174,9 +169,8 @@ function useLoadStory({ storyId, shouldLoad, restore, isDemo }) {
           autoAdvance: storyData?.autoAdvance,
           defaultPageDuration: storyData?.defaultPageDuration,
           backgroundAudio: storyData?.backgroundAudio,
-          embeddedTerms,
-          'story-tags': storyTags,
-          'story-categories': storyCategories,
+          taxonomies,
+          terms,
```

Comment:
Oh yup. this is a much better approach. Saves us from reading from then writing to the same story prop. lovely.